File: crates/cairn-resolver-eval/fixtures/kotlin/src/app/Service.kt
package app

import api.Greeter
import base.IntermediateService
import util.helper as aliasedHelper

class Service : IntermediateService(), Greeter {
    companion object {
        fun build(): Service = Service()
    }

    fun run() {
        this.greet()
        super.step()
        Service.Companion.build()
        aliasedHelper()
    }
}

File: crates/cairn-resolver-eval/fixtures/csharp/src/Service.cs
using Api;
using Lib;
using static Lib.Helpers;

namespace App;

public class Service : IntermediateService, IGreeter {
    public static Service Build() { return new Service(); }
    public void Greet() {}

    public void Run() {
        this.Greet();
        base.Step();
        Service.Build();
        RunHelper();
    }
}
